NST/src/main/java/com/ebay/runtime/RuntimeConfigManager.java

```diff
@@ -120,7 +120,7 @@ public PoolType getPoolType() {
 	 * 
 	 * @return True if schema validation should occur, false otherwise.
 	 */
-	public boolean validateSchema() {
+	public boolean getSchemaValidation() {
```

Comment:
@byarger-ebay I made this change so the method name is consistent with the other getters inside `RuntimeConfigManager`.

Reply:
Reverted in 6822efdb3fc404800c1684e52cc9e33e1925bc5f